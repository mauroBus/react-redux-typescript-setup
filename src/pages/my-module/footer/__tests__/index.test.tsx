import { shallow } from 'enzyme'
import React from 'react'
import { Footer } from '../'

describe('MyModule::Footer', () => {
  it('renders as a "footer" element', () => {
    const tree = shallow(<Footer />)
    expect(tree.type()).toBe('footer')
  })

  it('renders text message', () => {
    const tree = shallow(<Footer />)
    expect(tree.text()).toBe('This is a footer.')
  })

  it('renders', () => {
    const tree = shallow(<Footer />)
    expect(tree).toMatchSnapshot()
  })
})
