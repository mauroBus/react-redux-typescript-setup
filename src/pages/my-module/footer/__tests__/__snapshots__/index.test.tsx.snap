// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MyModule::Footer renders 1`] = `
<footer
  className="footer"
  role="contentinfo"
>
  This is a footer.
</footer>
`;
